packages/next-swc/crates/next-core/js/src/entry/config/next.js

```diff
@@ -20,6 +20,25 @@ const loadNextConfig = async (silent) => {
   nextConfig.exportPathMap = nextConfig.exportPathMap && {}
   nextConfig.webpack = nextConfig.webpack && {}
 
+  // Transform the `modularizeImports` option
+  nextConfig.modularizeImports = nextConfig.modularizeImports
```

Comment:
do we need to duplicate the logic here all the time? OO

Reply:
Yeah I should move it to somewhere else, maybe after loading the configuration. Will do that later!